HotelReservation/Business/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Text;
+using System.Threading.Tasks;
+using Business.DTO;
+using Business.Interfaces;
+using HotelReservation.Data;
+using HotelReservation.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Org.BouncyCastle.Crypto.Tls;
```

Comment:
dont use any references not from provided by Microsoft at this moment

Reply:
already deleted